### ISSUE TITLE
Wesnoth's 1.14 oldstable branch

### ISSUE THREAD

**Issue by loonycyborg** [flathub/flathub#5264]
Wesnoth's stable branch currently is 1.18 yet it's impossible to add 1.16 since custom branches aren't supported anymore. But there still exists 1.14 oldstable branch which probably should be removed for consistency.
```
$ flatpak --user remote-ls flathub | grep wesnoth
Битва за Веснот org.wesnoth.Wesnoth     1.14.17 1.14
Битва за Веснот org.wesnoth.Wesnoth     1.18.0  stable
```

**Comment by bbhtt** [comment]
branch/foo was always reserved for baseapps and extensions. This started being enforced recently.

**Comment by soliton-** [comment]
> branch/foo was always reserved for baseapps and extensions. This started being enforced recently.

I cannot tell what this means. Will the 1.14 branch get deleted at some point?

**Comment by bbhtt** [comment]
> Will the 1.14 branch get deleted at some point?
It will, the version that is published on that branch needs to be removed from the Flathub repository first. Then we can delete the Git branch.

**Comment by loonycyborg** [comment]
It seems that I also can't remove `branch/foo` branches from org.wesnoth.Wesnoth github repo because of branch protection rules.

**Comment by MXB2001** [comment]
Those of us with 1.14 installed via flatpak branch already won't be affected I assume? Or would an update attempt to delete my files? (always pays to be paranoid with this sort of auto-software)

**Comment by bbhtt** [comment]
well that branch hasn't recieved any update since the last 3 years. So you should switch.

But yes if it is removed from the server,your local copy will stay. At most it will display an EOL warning on command line.

**Comment by MXB2001** [comment]
> well that branch hasn't recieved any update since the last 3 years. So you should switch.

No, you misunderstand. I've got copies of 1.12, 1.14, 1.16 and 1.19 (the latest bleeding edge version) installed. Each is different enough to be its own game.
 
> But yes if it is removed from the server,your local copy will stay. At most it will display an EOL warning on command line.

Ah thanks, EOL is irrelevant to a guy who still runs 40 year old software on his 40 year old 8 bit PC's. : )
